cirrina/server.py

```diff
@@ -265,11 +265,11 @@ def _wrapper(func):
             return func
         return _wrapper
 
-    def enable_websockets(self):
+    def enable_websockets(self, location):
         """
         Enable websocket communication.
         """
-        self.app.router.add_route('GET', "/ws", self._ws_handler)
+        self.app.router.add_route('GET', location, self._ws_handler)
```

Comment:
As long as we do not support multiple websocket instances (which we don't want anyways) there will always be only one websocket route, so it makes sense to specify this route when enabling websockets.